test/polytopes.jl

```diff
@@ -54,6 +54,18 @@
     @test center(s) == P3(0.5, 0.5, 0.5)
     @test coordtype(center(s)) == T
 
+    # using Unitful.jl types
+    using Unitful: m
```

Comment:
We don't import modules inside the test files. You can do `x = T[0, 1]u"m"` to get the vector with units.